.github/workflows/api-kit-e2e-test.yml

```
@@ -21,4 +21,7 @@ jobs:
       - run: |
           yarn install --frozen-lockfile
           yarn build
```

Let's separate the steps so we split in shorter log chunks when we check the action.

```suggestion
      - name: Yarn install
        run: yarn install --frozen-lockfile

      - name: Build
        run: yarn build

      - name: Test
        run: |
```